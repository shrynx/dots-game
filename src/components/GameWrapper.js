// @flow
import styled from 'styled-components';

const GameWrapper = styled.div`
  width: 100%;
  margin: 0 auto;
  padding: 0;
`;

export default GameWrapper;
