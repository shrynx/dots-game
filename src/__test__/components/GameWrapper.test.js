import React from 'react';
import { render } from 'react-testing-library';
import GameWrapper from '../../components/GameWrapper';

it('GameWrapper component should render correctly', () => {
  const { container } = render(<GameWrapper />);
  expect(container).toMatchSnapshot();
});
