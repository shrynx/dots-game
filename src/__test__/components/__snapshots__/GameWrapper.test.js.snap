// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameWrapper component should render correctly 1`] = `
<div>
  <div
    class="sc-bdVaJa jlBtQz"
  />
</div>
`;
